launcher/ui/pages/modplatform/atlauncher/AtlListModel.cpp

```diff
@@ -66,7 +66,7 @@ QVariant ListModel::data(const QModelIndex &index, int role) const
         auto icon = APPLICATION->getThemedIcon("atlauncher-placeholder");
 
         auto url = QString(BuildConfig.ATL_DOWNLOAD_SERVER_URL + "launcher/images/%1.png").arg(pack.safeName.toLower());
-        ((ListModel *)this)->requestLogo(pack.safeName, url);
+        const_cast<ListModel*>(this)->requestLogo(pack.safeName, url);
```

Comment:
Another rabbit hole comment for future reference. Very similar to the ModModel.cpp one.